### PR TITLE
kuncao0228- Added extractText.py and testExtractTest.py to capitalize…

### PR DESCRIPTION
**Pull request by kuncao0228**
Variable and Equation Extraction

### CODE REVIEW

File: bin/sir_model_intro.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (jpfairbanks) [inline bin/sir_model_intro.md]: I think this needs to be in /examples somewhere.

**Comment by kuncao0228** [comment]
Added more tests, provided a output/input directories for the extracted texts, handles headers

**Comment by kuncao0228** [comment]
removed .pyc and added the require.txt, 
added in the csv output and text inputs with extracted texts
sovMain.py runs with an input argument of the dir_path of the extracted text location

**Comment by crherlihy** [comment]
Just a note that if hitting the Rosette api becomes restrictive at some point, we can consider using spaCy and/or textacy for pos tagging/extraction.

**Comment by kuncao0228** [comment]
My recent pull request is no longer using rosette just spaCy

**Comment by crherlihy** [comment]
@kuncao0228 great! sorry, am getting caught up.

**Pull request merged**